tests/integration/components/permission-language-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('permission-language', 'Integration | Component | permission language', {
+  integration: true
+});
+
+test('it renders', function(assert) {
```

Comment:
Would it be possible to add tests to ensure that, if we're using a particular model, we're getting the proper text? We don't have a lot of tests in ember yet, and we're not quite all-in with knowing the best way to test, but that seems like a worthwhile set of tests. It might also be worth verifying, if possible, that for every model we have copyright text to go with. Maybe ask Nici if that's an assumption we should make.